### ISSUE TITLE
Develop workflow for processing training data set

### ISSUE THREAD

**Issue by fmahnke** [spectrumbranch/retro-translation-project#5]
Scope the work to start with a corpus of videos and end with ground truth for tesseract

I'm guessing we'd take the video, run it through `mortar.video` somehow, then `mortar.pipeline` ?

which would create a large basis of "line image and starter label" pairs, which then need to be validated/corrected somehow before being used for training

- [ ] add capability to easily generate training data from any subset of our reference images/translations
- [ ] add capability to determine the accuracy achieved by running any subset back through Tess against generated training data
